### DISCUSSION TITLE
Average Electricity Cost (Mid Case vs. Cambium)

### DISCUSSION THREAD

**Discussion by astronobri** [NREL/ReEDS-2.0#113] [Q&A: Data]
Hi, I ran the Standard Scenario Mid Case and I found this result puzzling -- VA, WV, and PA should have similar costs for electricity, in agreement with your Cambium 2021 results. Do you have the cases.csv file for your Cambium data, or is there a way to correct the Mid Case for this? Thanks!

![avg_elec_cost_va_pa_wv](https://user-images.githubusercontent.com/29382951/162279172-f73a2aad-5766-43e4-908e-aeb919134508.PNG)

**Comment by wesleyjcole** [comment]
These values don't look correct for average electricity costs.  $1200/MWh is really expensive.

ReEDS includes electricity price outputs using the Bokeh Viewer tool.  You can check out https://www.youtube.com/watch?v=8Xi59M4bB6I to learn more there.

**Reply by astronobri** [reply]
<img width="1058" alt="Screen Shot 2022-04-26 at 3 30 03 PM" src="https://user-images.githubusercontent.com/29382951/165377564-44c65b8e-0f8c-462e-95a8-8883a537595f.png">

This is what I'm getting when using Bokehpivot to plot the BA-level Avg Electricity costs for the Mid_Case in cases_standardscenarios.csv in the ReEDS repository. This doesn't look right for the Mid Case -- any idea what could be going on?

**Reply by vcelloho** [reply]
What does the capacity and generation look like for the three selected states? I'm curious what is being built and operated.

**Reply by astronobri** [reply]
![midcase_generation_va_pa_wv](https://user-images.githubusercontent.com/29382951/165400848-4e42f8d5-7a44-46fe-bd05-5d4d1d08ac1d.PNG)
![midcase_capacity_va_pa_wv](https://user-images.githubusercontent.com/29382951/165400853-441f9f31-fe14-483e-bdeb-95a2e9326218.PNG)

**Reply by astronobri** [reply]
I'm not sure that I see anything in the capacity or generation that would cause such high prices? Any thoughts on how to diagnose from here?

**Reply by vcelloho** [reply]
Okay so there's a couple things that I can think of to test. The first thing would be to check the ReEDS PCA region level electricity prices. Instead of of exploding on the state explode on the ReEDS region, in the drop downs this should be rb. Then change the filter to just be the state of Virginia. The goal here is to see if one ReEDS region or the entire state is behaving strangely.

I'd also be interested in seeing the comparison with the wholesale electricity prices Requirement Prices and Quantities BA -> Bulk System Electricity Price. Is the same behavior present here.

It looks to me from the average prices VA is exporting energy and capacity while experiencing high prices, I'm trying to think of why this would be.
